urdfenvs/urdf_common/urdf_env.py

```diff
@@ -247,8 +247,15 @@ def step(self, action):
 
         if self._render:
             self.render()
-        return ob, reward, self._done, self._info
 
+        terminated = self._done
+        truncated = self._getTruncated()
+        return ob, reward, terminated, truncated, self._info
+    
+    def _getTruncated(self) -> bool:
```

Comment:
Can you respect the pep guidelines for method naming, this should be 'get_truncated'. Can you check it for the other functions as well.

Reply:
Ok.